bemserver_ui/static/scripts/modules/views/timeseries/dataExplore.js

```diff
@@ -65,11 +65,77 @@ export class TimeseriesDataExploreView {
         this.#endDatetimePickerElmt.dateMin = this.#startDatetimePickerElmt.date;
     }
 
+    #addTsParamInputs(tsData) {
+        let tsParam = document.createElement("div");
+        tsParam.id = "tsParam-" + tsData.itemId;
+        
+        tsParam.innerHTML =
+        "<div class=\"row d-xl-flex d-grid m-2 mb-3\">"
+        + "<h6>"
+        + tsData.itemName
+        + "</h6>"
+        + "<div class=\"col pb-2 pb-xl-0\">"
+        + "<div class=\"input-group input-group-sm\">"
+        + "<span class=\"input-group-text\">Position</span>"
+        + "<select class=\"form-select\" id=\"position-"
+        + tsData.itemId
+        + "\" name=\"position\" aria-label=\"Select a position\">"
+        + "<option value=\"left\" selected>Left</option>"
+        + "<option value=\"right\">Right</option>"
+        + "</select>"
+        + "</div>"
+        + "</div>"
+        + "<div class=\"col pb-2 pb-xl-0\">"
+        + "<div class=\"input-group input-group-sm\">"
+        + "<span class=\"input-group-text\">Type</span>"
+        + "<select class=\"form-select\" id=\"type-"
+        + tsData.itemId
+        + "\" name=\"type\" aria-label=\"Select a type of graph\">"
+        + "<option value=\"line\" selected>Line chart</option>"
+        + "<option value=\"bar\">Bar chart</option>"
+        + "</select>"
+        + "</div>"
+        + "</div>"
+        + "<div class=\"col pb-2 pb-xl-0\">"
+        + "<div class=\"input-group input-group-sm\">"
+        + "<span class=\"input-group-text\">Style</span>"
+        + "<select class=\"form-select\" id=\"style-"
+        + tsData.itemId
+        + "\" name=\"style\" aria-label=\"Select a style\">"
+        + "<option value=\"solid\" selected>Solid</option>"
+        + "<option value=\"dashed\">Dashed</option>"
+        + "<option value=\"dotted\">Dotted</option>"
+        + "</select>"
+        + "</div>"
+        + "</div>"
+        + "<div class=\"col pb-2 pb-xl-0\">"
+        + "<div class=\"input-group input-group-sm\">"
+        + "<span class=\"input-group-text\">Color</span>"
+        + "<input type=\"color\" class=\"form-control form-control-color\" id=\"color-"
+        + tsData.itemId
+        + "\" name=\"color\" value=\""
+        + this.#chart.colors[(this.#tsSelector.selectedItemNames.length - 1) % (this.#chart.colors.length)]
+        + "\">"
+        + "</div>"
+        + "</div>";
+
+        document.getElementById("tsParam").appendChild(tsParam);
```

Comment:
We could defined a `#tsParamsContainerElmt` view class attribute to cache (in `#cacheDOM` function) the `tsParam` div element, so we have always within reach (and without search) the HTML element instance to add timeseries param inputs.